### ISSUE TITLE
(Master Alpha) RDImport MySQL Error with Dropbox

### ISSUE THREAD

**Issue by deltecent** [ElvishArtisan/rivendell#329]
Investigating...

```
Nov 30 16:55:24 rdhost rdxport.cgi: unlink(/var/snd/060501_001.wav): Success
Nov 30 16:55:24 rdhost rdimport: invalid SQL or failed DB connection[Duplicate entry '60501' for key 'PRIMARY' QMYSQL: Unable to execute query]: insert into CART set NUMBER=60501,TYPE=1,GROUP_NAME="PODCASTS",TITLE="[cart 060501-1]"
```

**Comment by deltecent** [comment]
This error is generated when a dropbox specifies a specific cart. RDImport calls ```RDCart::create()``` on the existing cart so the ```insert``` fails with ```Duplicate entry 'XXXXX' for key 'PRIMARY'```.

RDImport should probably only call ```RDCart::create()``` if the cart doesn't already exist.

**Comment by ElvishArtisan** [comment]
This was swatted in [stable 586661c]. We just need to forward port it to 'master'.

&lt;sigh&gt;. The joys of maintaining two branches concurrently...

**Comment by ElvishArtisan** [comment]
Fixed in [master  d2fc3f9].
